src/humancellatlas/data/metadata/api.py

```diff
@@ -107,22 +107,68 @@ def __init__(self, entity: LinkedEntity, other_entity: Entity, forward: bool) ->
                          other_entity.address)
 
 
+@dataclass(frozen=True)
+class ProjectPublication:
+    publication_title: str
+    publication_url: str
+
+    @staticmethod
+    def from_json(json: JSON) -> None:
+        return ProjectPublication(
+            publication_title=json['publication_title'],
+            publication_url=json.get('publication_url')
+        )
+
+
+@dataclass(frozen=True)
+class ProjectContact:
+    contact_name: str
+    email: str
+    institution: str
+    laboratory: str
+    corresponding_contributor: bool
+
+    @staticmethod
+    def from_json(json: JSON) -> None:
+        return ProjectContact(
+            contact_name=json['contact_name'],
+            email=json.get('email'),
+            institution=json['institution'],
+            laboratory=json.get('laboratory'),
+            corresponding_contributor=json.get('corresponding_contributor', False)
```

Comment:
Where in the schema does it say that False should be assumed if field is missing?

Reply:
I couldn't find it. IOW, default should be None, like the other optional fields.